docs/core/tutorials/using-with-xplat-cli.md

```diff
@@ -5,7 +5,7 @@ keywords: .NET Core, CLI
 author: cartermp
 ms.author: mairaw
 ms.date: 03/08/2017
-ms.topic: article
+ms.topic: get-started-article
```

Comment:
are you sure that's the value? I don't see that in the spreadsheet that lists all possible values. I also don't see article in there. I'll follow up with Katya on that.